apps/mudita-center-e2e/src/specs/stress-tests/device-drawer-stress-test.ts

```diff
@@ -0,0 +1,248 @@
+      })
+
+      await browser.pause(6000)
+    }
+  })
+
+  it("Verify Devices 4,5 and 6 on Drawer and check their Serial Numbers, scroll down and select 6th Kompakt", async () => {
+    const deviceSelectDrawer = await drawerPage.deviceSelectDrawer
+    await expect(deviceSelectDrawer).toBeDisplayed()
+
+    const deviceImageOnDrawer = await drawerPage.deviceImageOnDrawer
+    await expect(deviceImageOnDrawer).toBeDisplayed()
+
+    const fourthDeviceOnDrawer = await drawerPage.getDeviceOnDrawer(
+      fourthSerialNumber
+    )
+    const fifthDeviceOnDrawer = await drawerPage.getDeviceOnDrawer(
+      fifthSerialNumber
+    )
+    const sixthDeviceOnDrawer = await drawerPage.getDeviceOnDrawer(
+      sixthSerialNumber
+    )
+    await expect(fourthDeviceOnDrawer).toBeDisplayed()
+    await expect(fifthDeviceOnDrawer).toBeDisplayed()
+    await expect(sixthDeviceOnDrawer).toBeDisplayed()
+
+    const drawerSerialNumbers = await drawerPage.drawerDeviceSerialNumber
+    const drawerDeviceNames = await drawerPage.drawerDeviceName
+
+    const devices = [
+      { serialNumber: fourthSerialNumber, name: "Kompakt", index: 3 },
+      { serialNumber: fifthSerialNumber, name: "Kompakt", index: 4 },
+      { serialNumber: sixthSerialNumber, name: "Kompakt", index: 5 },
+    ]
+
+    for (const device of devices) {
+      const deviceSerialNumber = await drawerSerialNumbers[
+        device.index
+      ].getText()
+      await expect(deviceSerialNumber).toEqual(device.serialNumber)
+
+      const deviceName = await drawerDeviceNames[device.index]
+      await expect(deviceName).toHaveText(device.name)
+    }
+
+    await browser.execute(() => {
+      window.scrollBy(0, 500)
+    })
+
+    await browser.pause(1000)
+
+    await sixthDeviceOnDrawer.waitForClickable()
+    await sixthDeviceOnDrawer.click()
+  })
+
+  it("Verify 6th Kompakt with new Serial Number and click Show connected devices", async () => {
+    const serialNumberLabel = await OverviewKompaktPage.serialNumberLabel
+    const serialNumberValue = await OverviewKompaktPage.serialNumberValue
+    await expect(serialNumberLabel).toHaveText("Serial number")
+    await expect(serialNumberValue).toHaveText(
+      overviewDataWithOneSimCard6th.summary.about.serialNumber.text.toString()
+    )
+    const selectConnectedDevices = await overviewPage.selectConnectedDevices
+    await selectConnectedDevices.waitForClickable()
+    await expect(selectConnectedDevices).toHaveText("6")
+    await selectConnectedDevices.click()
+  })
+
+  it("Disconnect the devices and check if Welcome Page is present", async () => {
+    E2EMockClient.removeDevice("path-1")
+    E2EMockClient.removeDevice("path-2")
+    E2EMockClient.removeDevice("path-3")
+    E2EMockClient.removeDevice("path-4")
+    E2EMockClient.removeDevice("path-5")
+    E2EMockClient.removeDevice("path-6")
+
+    const homeHeader = await HomePage.homeHeader
+    await homeHeader.waitForDisplayed()
```

Comment:
🔴  mockdev.51
```
[0-0] Error in "Kompakt switching devices.Disconnect the devices and check if Welcome Page is present"
Error: element ("h1[data-testid='onboarding-title-header']") still not displayed after 6000ms
```

Reply:
[chrome 116.0.5845.228 windows #0-0]
[chrome 116.0.5845.228 windows #0-0] » \src\specs\stress-tests\device-drawer-stress-test.ts
[chrome 116.0.5845.228 windows #0-0] Kompakt switching devices
[chrome 116.0.5845.228 windows #0-0]    ✓ Connect 1,2 Kompakt devices
[chrome 116.0.5845.228 windows #0-0]    ✓ Verify Devices 1 and 2 on Drawer and check their Serial Numbers
[chrome 116.0.5845.228 windows #0-0]    ✓ Connect 3rd Kompakt device
[chrome 116.0.5845.228 windows #0-0]    ✓ Verify Device 3 on the Drawer and check the Serial Number
[chrome 116.0.5845.228 windows #0-0]    ✓ Connect 4,5,6 Kompakt devices
[chrome 116.0.5845.228 windows #0-0]    ✓ Verify Devices 4,5 and 6 on Drawer and check their Serial Numbers, scroll down and select 6th Kompakt
[chrome 116.0.5845.228 windows #0-0]    ✓ Verify 6th Kompakt with new Serial Number and click Show connected devices
[chrome 116.0.5845.228 windows #0-0]    ✓ Disconnect the devices and check if Welcome Page is present
[chrome 116.0.5845.228 windows #0-0]
[chrome 116.0.5845.228 windows #0-0] 8 passing (42.7s)


Spec Files:      1 passed, 1 total (100% completed) in 00:00:45

mockdev 51